### PR TITLE
585 card list groups

### PR DESCRIPTION
**Pull request by Sudalius**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class CardListGroups extends Section {
+
+    @UI(".card-header") public Label cardHeader;
+    @UI(".list-group-item") public List<WebElement> listGroups;
+}
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java]: no newline at the end of file

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class CardListGroups extends Section {
+
+    @UI(".card-header") public Label cardHeader;
+    @UI(".list-group-item") public List<WebElement> listGroups;
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java]: it's better to use our class WebList

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    public void checkCardListGroupsValues() {
+        int checkedValues = 0;
+        for (WebElement s:cardListGroups.listGroups) {
+            if (cardListGroupsValues.contains(s.getText())) {
+                checkedValues++;
+           }
+        }
+        assertEquals(cardListGroupsValues.size(), checkedValues);
+    }
+}
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java]: no new line at the end of file

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+    @Test
+    public void checkCardListCellsQuantity() {
+        assertEquals(cardListGroups.listGroups.size(), cardListGroupsSize);
+    }
+
+    @Test
+    public void checkCardListGroupsValues() {
+        int checkedValues = 0;
+        for (WebElement s:cardListGroups.listGroups) {
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java]: please use auto reformatting in intellij idea (ctrl-alt-L / cmd-alt-L)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            if (cardListGroupsValues.contains(s.getText())) {
+                checkedValues++;
+           }
+        }
+        assertEquals(cardListGroupsValues.size(), checkedValues);
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java]: list.assertThat().size(size) (if weblist)

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java (hunk 1 above)
Comment (Sudalius) [reply jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java]: fixed

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java (hunk 2 above)
Comment (Sudalius) [reply jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/CardListGroups.java]: fixed

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java (hunk 3 above)
Comment (Sudalius) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java]: fixed

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java (hunk 4 above)
Comment (Sudalius) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java]: fixed

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java (hunk 5 above)
Comment (Sudalius) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardListGroupsTests.java]: In this class I can't use WebList, cause I need to initialize it with 3 strings

**Pull request merged**